### PR TITLE
Configure the Semantic Logger to output JSON

### PR DESCRIPTION
**Pull request by vacabor**
As per https://technical-guidance.education.gov.uk/infrastructure/monitoring/logit/#ruby-on-rails.

Extracted from #2815.

### CODE REVIEW

**Comment by AbigailMcP** [comment]
> Note: We could also remove `filebeat` at this point.

Tracked here: https://dfedigital.atlassian.net/jira/software/projects/CAPT/issues/CAPT-1730?jql=project%20%3D%20%22CAPT%22%20AND%20text%20~%20%22filebeat%22%20ORDER%20BY%20created%20DESC

**Comment by AbigailMcP** [comment]
Once we're happy the logs are working as they should, let's update [these docs](https://github.com/DFE-Digital/claim-additional-payments-for-teaching/blob/master/docs/logging.md) too.

**Review by AbigailMcP — APPROVED** [review]

File: config/environments/production.rb [hunk 1]
```diff
@@ -47,8 +47,10 @@
   # information to avoid inadvertent exposure of personally identifiable information (PII).
   config.log_level = :info
 
-  # Prepend all log lines with the following tags.
-  config.log_tags = [:request_id]
+  # https://technical-guidance.education.gov.uk/infrastructure/monitoring/logit/#ruby-on-rails
```
Comment (saliceti) [inline config/environments/production.rb:50]: Could you test with and without dfe analytics? In other services we saw it was flooding the logs and we filtered them out using:
https://github.com/DFE-Digital/teaching-vacancies/blob/main/config/environments/production.rb#L77

File: terraform/application/config/production.tfvars.json [hunk 2]
```diff
@@ -18,5 +18,6 @@
   },
   "enable_monitoring": true,
   "statuscake_contact_groups": [195955, 282453],
-  "external_url": "https://claim-additional-payments-for-teaching-production-web.teacherservices.cloud/healthcheck"
+  "external_url": "https://claim-additional-payments-for-teaching-production-web.teacherservices.cloud/healthcheck",
+  "enable_logit": true
```
Comment (saliceti) [inline terraform/application/config/production.tfvars.json:22]: Can we enable first on review and test? So we can validate before shipping prod logs

**Comment by saliceti** [comment]
[Claims: onboard to Logit.io](https://trello.com/c/OgNRUOZm/1881-claims-onboard-to-logitio)

**Pull request merged**